packages/cli-doctor/src/tools/healthchecks/xcodeEnv.ts

```diff
@@ -30,10 +29,19 @@ export default {
   getDiagnostics: async (_, config) => {
     try {
       const projectRoot = config?.root ?? findProjectRoot();
-      const missingXcodeEnvFile = findPodfilePaths(projectRoot).some((p) => {
-        const basePath = path.dirname(p);
-        return !pathHasXcodeEnvFile(basePath);
-      });
+
+      const iosFolderPath = config?.project.ios
+        ? config?.project.ios.sourceDir
+        : path.join(projectRoot, 'ios');
```

Comment:
`sourceDir` is set to $root/ios even if it's not explicitly defined

Reply:
Correct!